src/pages/login.tsx

```diff
@@ -0,0 +1,43 @@
+import { type NextPage } from "next";
+import React from "react";
+import { Layout } from "../components/Layout";
+import { LoginForm } from "../components/LoginForm";
+import { useAuth } from "../hooks/useAuth";
+import { useToast } from "../hooks/useToast";
+import { AuthProvider } from "../providers/AuthProvider";
+import { LoginValues } from "../schemas/loginSchema";
+import { api } from "../utils/api";
+import Router from "next/router";
+
+const LoginPage: NextPage = ({}) => {
+  const authData = useAuth();
```

Comment:
I think you can just destructure it

```
const { login } = useAuth();
```

Or rename it. Try to never have `data` inside a variable name, because it can mean anything.

```
const auth = useAuth()
```

would be much better